src/py-opentimelineio/opentimelineio-bindings/otio_utils.cpp

```diff
@@ -40,9 +40,9 @@ py::object plain_int(int i) {
 
 py::object plain_int(int64_t i) {
     #if PY_MAJOR_VERSION >= 3
```

Comment:
looking good! can you remove the version check as it is now moot?

Reply:
Good point; this is maybe getting a bit off track but the other plain_int(int) function above could probably also use the same change since apparently PyInt and PyLong seem to be interchangeable?
https://www.python.org/dev/peps/pep-0237/

Reply:
yes, that would be consistent. I think that would put a nice bow on this commit, and then I can go ahead and merge. Thanks for the detective work on the merge failure, that was mighty mysterious given the lack of clear output from CI.

Reply:
Sorry about tox eating the message like that.  We've been talking offline about what we might be able to do to improve the CI results.